src/locales/cy/translation.json

```diff
@@ -44,7 +48,7 @@
       "label": "Parhau"
     },
     "contactAccounts": {
-      "contactLinkHref": "https://signin.account.gov.uk/contact-us",
+      "contactLinkHref": "https://signin.account.gov.uk/support",
```

Comment:
I couldn't find usage for these anywhere in the code base. Are they still needed?

Reply:
Thanks James - I've just pushed a commit to removes these 👍🏽